src/main/resources/application.properties

```diff
@@ -11,7 +11,7 @@ logging.level.org.apache.kafka = WARN
 spring.jpa.database-platform=org.hibernate.dialect.MySQL5Dialect
 spring.datasource.password=${DB_PASSWORD}
 spring.datasource.username=${DB_USER}
-spring.datasource.url=jdbc:mysql://${DB_HOST:localhost:3306}/${DB_NAME}
```

Comment:
This was intended, since it is our default value

Reply:
So is this how you declare a default value for DB_HOST if there is no value present? I didn't know that, will revert this then.

Reply:
Yes, exactly